mollie/api/objects/order_line.py

```diff
@@ -0,0 +1,108 @@
+from .base import Base
+
+
+class OrderLine(Base):
+
+    def __init__(self, data, resource=None, client=None):
+        """
+        Override the super __init__ to assign the Client to the result object, which is more flexible since it's
+        not tied to a single API resource type
+        """
+        super(OrderLine, self).__init__(data, None)
+        self.client = client
+
+    @classmethod
+    def get_object_name(cls):
+        return 'lines'
+
+    @property
+    def id(self):
+        return self._get_property('id')
+
+    @property
+    def resource(self):
+        return self._get_property('resource')
+
+    @property
+    def order_id(self):
+        return self._get_property('orderId')
+
+    @property
+    def type(self):
+        return self._get_property('type')
+
+    @property
+    def name(self):
+        return self._get_property('name')
+
+    @property
+    def status(self):
```

Comment:
Which statuses are known? Also I'm not seeing additional methods like `OrderLine.is_created()` etc?